pkg/entities/alerts.go

```diff
@@ -177,3 +218,47 @@ func (a *StateHashAlert) Time() time.Time {
 func (a *StateHashAlert) String() string {
 	return fmt.Sprintf("%s: %s", a.Type(), a.Message())
 }
+
+func (a *StateHashAlert) ID() string {
```

Comment:
Why should `ID` be that complex?

Reply:
Yep, because we need to identify each alert properly.